token-lending/cli/README.md

```diff
@@ -0,0 +1,129 @@
+# SPL Token Lending program command line interface
+
+A basic CLI for initializing lending markets and reserves for SPL Token Lending.
+See https://spl.solana.com/token-lending for more details
+
+## Deploy a lending program on Solana
+
+1. [Install the Solana CLI](https://docs.solana.com/cli/install-solana-cli-tools)
+
+1. Clone the SPL repo:
+   ```shell
+   git clone https://github.com/solana-labs/solana-program-library.git
+   ```
+
+1. Go to the new directory:
+   ```shell
+   cd solana-program-library
+   ```
+
+1. Generate a keypair for yourself:
+   ```shell
+   solana-keygen new -o owner.json
+
+   # Wrote new keypair to owner.json
+   # ================================================================================
+   # pubkey: F5242QU7NC4jM8yjqqiv8pZRjL9K8229EBTXHBYtSAo5
+   # ================================================================================
+   # Save this seed phrase and your BIP39 passphrase to recover your new keypair:
+   # your seed words here never share them not even with your mom
+   # ================================================================================
+   ```
+   This pubkey will be the owner of the lending market that can add reserves to it.
+
+1. Generate a keypair for the program:
+   ```shell
+   solana-keygen new -o lending.json
+
+   # Wrote new keypair to lending.json
+   # ============================================================================
+   # pubkey: 25MruiXVk27KoQPNKpMr44UDPHPCcSpRBfhXcgMDTwqQ
+   # ============================================================================
+   # Save this seed phrase and your BIP39 passphrase to recover your new keypair:
+   # your seed words here never share them not even with your mom
+   # ============================================================================
+   ```
+   This pubkey will be your Program ID.
+
+1. Open `./token-lending/program/src/lib.rs` in your editor. In the line
+   ```rust
+   solana_program::declare_id!("TokenLending11111111111111111111111111111111");
+   ```
+   replace `TokenLending11111111111111111111111111111111` with your Program ID, e.g.:
+   ```rust
+   solana_program::declare_id!("25MruiXVk27KoQPNKpMr44UDPHPCcSpRBfhXcgMDTwqQ");
```

Comment:
How about deploying a new version of the lending program to devnet and testnet and updating the declared id in source? Then, devs can optionally override with their own deployed lending program with `--lending-program-id` in the cli

Reply:
Done.

Reply:
The lending program has been deployed to devnet and testnet, and a lending market has been added to both. A reserve has been added to devnet. However, the testnet program cannot have reserves added, as the Pyth program is not available outside of devnet yet.

Reply:
No problem, devnet is good enough for now, thanks!